Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -26,13 +26,15 @@ final class EvaluationInteractorImpl: EvaluationInteractor {
     let evaluationDao: EvaluationDao
     let defaults: Defaults
     let idGenerator: IdGenerator
+    let config: BKTConfig
     let logger: Logger?
 
-    init(apiClient: ApiClient, evaluationDao: EvaluationDao, defaults: Defaults, idGenerator: IdGenerator, logger: Logger? = nil) {
+    init(apiClient: ApiClient, evaluationDao: EvaluationDao, defaults: Defaults, idGenerator: IdGenerator, config: BKTConfig, logger: Logger? = nil) {
```

Comment:
If we only need the `featureTag`, please pass the `featureTag` instead of `BKTConfig`.

Reply:
Fixed https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/b59973843dc5ef353f8cf8cb71b64f7655da4b5f